templates/public-page.html

```diff
@@ -0,0 +1,88 @@
+{% extends "auth.html" %}
+
+{% block main %}
+    <div id="general_info" class="flex flex-row items-center h-64 mt-8 mx-2">
+        <div class="w-1/3 mr-4 h-full flex items-center justify-center">
+            {% if user_info.image %}
+                <img src="{{static('/images/profile_images/' + user_info.image + '.png')}}" class="w-64">
+            {% else %}
+                <img src="{{static('/images/Hedy-logo.png')}}" class="w-64">
+            {% endif %}
+        </div>
+        <div class="w-full h-full bg-blue-200 flex flex-col rounded-lg border border-black">
+            <div class="flex flex-row items-center h-12 bg-blue-600 rounded-t-lg px-4 font-medium">
+                <div class="text-white tracking-wide text-4xl">{{ user_info.username }}</div>
+                {% if user_achievements['achieved'] %}
+                    <div class="text-white text-xl ml-auto">{{ user_achievements['achieved']|length }} {{ auth.achievements }}</div>
```

Comment:
Maybe we should be using auth.my_achievements here and not auth.achievements?
![Screenshot 2022-01-24 at 12 20 31](https://user-images.githubusercontent.com/3010723/150765014-2e157794-120c-4180-9916-9ea646f38e0a.png)

Reply:
I think also a merge error as this should be `auth.my_achievements` already, but fixed!